docs/component.ipynb

```diff
@@ -22,11 +22,7 @@
   },
   {
    "cell_type": "raw",
-   "metadata": {
-    "deletable": true,
-    "editable": true,
-    "raw_mimetype": "text/restructuredtext"
-   },
+   "metadata": {},
```

Comment:
It looks like you've committed all doc notebooks after you executed them locally. Did you actually made edits to the docs?

Reply:
It looks like it although I don't remember executing this particular notebook. I sure did not change anything in there. 
Is there anything I should do about that?

Reply:
No edits, no

Reply:
Ok, then reset the docs files with `git checkout -- filename`

Reply:
I came back 8 steps on those